test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectBasisForClaimControllerSpec.scala

```diff
@@ -262,6 +263,7 @@ class SelectBasisForClaimControllerSpec
         inSequence {
           mockAuthWithNoRetrievals()
           mockGetSession(session.copy(journeyStatus = Some(updatedJourney)))
```

Comment:
there should be some tests to show the different sets of basis of claims for the different journey types - I think that is the purpose of this ticket

Reply:
That's taken care of in BasisOfClaimsSpec.